### PR TITLE
Macos ci update

### PR DESCRIPTION
**Pull request by PatTheMav**

### CODE REVIEW

File: CI/macos/package/packageApp.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+rm -f -r ./OBS.app/Contents/Frameworks/QtNetwork.framework
+
+# put qt network in here because streamdeck uses it
+cp -R /usr/local/opt/qt/lib/QtNetwork.framework ./OBS.app/Contents/Frameworks
+chmod -R +w ./OBS.app/Contents/Frameworks/QtNetwork.framework
+rm -r ./OBS.app/Contents/Frameworks/QtNetwork.framework/Headers
+rm -r ./OBS.app/Contents/Frameworks/QtNetwork.framework/Versions/5/Headers/
+chmod 644 ./OBS.app/Contents/Frameworks/QtNetwork.framework/Versions/5/Resources/Info.plist
+install_name_tool -id @executable_path/../Frameworks/QtNetwork.framework/Versions/5/QtNetwork ./OBS.app/Contents/Frameworks/QtNetwork.framework/Versions/5/QtNetwork
+install_name_tool -change /usr/local/Cellar/qt/5.14.1/lib/QtCore.framework/Versions/5/QtCore @executable_path/../Frameworks/QtCore.framework/Versions/5/QtCore ./OBS.app/Contents/Frameworks/QtNetwork.framework/Versions/5/QtNetwork
```
Comment (snicolai-blog) [inline CI/macos/package/packageApp.sh:60]: Should this be ${QT_VERSION:-5.14.1} ?

File: CI/macos/package/app/Info.plist [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	<key>CFBundleGetInfoString</key>
+	<string>OBS - Free and Open Source Streaming/Recording Software</string>
+	<key>CFBundleExecutable</key>
+	<string>obs</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.obsproject.obs-studio</string>
+	<key>CFBundlePackageType</key>
+	<string>APPL</string>
+	<key>LSMinimumSystemVersion</key>
+	<string>10.8.5</string>
```
Comment (snicolai-blog) [inline CI/macos/package/app/Info.plist:18]: CEF was built with 10.11, does it really run on a system this old?

File: CI/macos/package/packageApp.sh (hunk 1 above)
Comment (PatTheMav) [reply CI/macos/package/packageApp.sh:60]: Yep, will amend the commit.

File: CI/macos/package/app/Info.plist (hunk 2 above)
Comment (PatTheMav) [reply CI/macos/package/app/Info.plist:18]: Good point, I just copied the old `plist` over, but will change this as well.

**Pull request closed without merging**